assets/components/pages/project/curate/VariantData.js

```diff
@@ -187,7 +422,7 @@ class VariantData extends React.Component {
         </List.Item>
         <List.Item>
           <strong>Allelic Depths:</strong>{" "}
-          {variant.AD ? variant.AD.map((depths) => `(${depths.join(", ")})`).join(", ") : "N/A"}
+          {(variant.AD ?? []).map((depths) => `(${depths.join(", ")})`).join(", ") || "N/A"}
```

Comment:
This optional coalescing is quite modern (>2020 browsers) - would be good to check the compiled bundle to check if this is being transpiled away.

Reply:
In the compiled bundle, this line is compiled as 
`
t.createElement("strong",null,"Allelic Depths:")," ",(null!==(o=u.AD)&&void 0!==o?o:[]).map((function(e){return"(".concat(e.join(", "),")")})).join(", ")||"N/A")
`

Reply:
Dope! Thanks for checking :)